src/assets/css/app.scss

```diff
@@ -1 +1,2 @@
 @import "panels";
+@import "../../../node_modules/highlight.js/styles/idea";
```

Comment:
Perhaps we should make a separate file called `_hljs.scss` og `_highlightjs.scss` and import that in app.scss, since I assume there will be more regarding highlighting.

Also, don't forget the newline at the end of the file ;)